src/api/tralis/TralisAPI.js

```diff
@@ -56,9 +56,7 @@ class TralisAPI {
     }
     /** @ignore */
     this.conn = new WebSocketConnector(wsUrl);
-    /** @ignore */
-    this.dfltProjection = 'epsg:3857';
-    this.conn.setProjection(this.dfltProjection);
+    this.conn.setProjection(options.projection || 'epsg:3857');
```

Comment:
I guess we should add `options.projection` to the `constructor` JSDoc.

Reply:
Done